hudi-aws/src/main/java/org/apache/hudi/aws/sync/AWSGlueCatalogSyncClient.java

```diff
@@ -89,16 +90,19 @@ public class AWSGlueCatalogSyncClient extends HoodieSyncClient {
   private static final Logger LOG = LoggerFactory.getLogger(AWSGlueCatalogSyncClient.class);
   private static final int MAX_PARTITIONS_PER_REQUEST = 100;
   private static final long BATCH_REQUEST_SLEEP_MILLIS = 1000L;
+  private static final String ENABLE_MDT_LISTING = "hudi.metadata-listing-enabled";
```

Comment:
Is this based on Presto or Trino connector configuration?  How does it enable Athena to use the metadata table for file listing? Could you add docs and references here?

Reply:
All details in jira

Reply:
@parisni Let's add the javadocs here in the code as well.

Reply:
@yihua added details